graphql-dgs-spring-graphql/src/main/kotlin/com/netflix/graphql/dgs/apq/DgsAPQSupportAutoConfiguration.kt

```diff
@@ -105,8 +109,13 @@ open class DgsAPQSupportAutoConfiguration {
         }
     }
 
+    // We want this version only if there is no micrometer meter registry
     @Configuration
-    @ConditionalOnMissingBean(APQMicrometerMeteredCaffeineCacheConfiguration::class)
```

Comment:
Still want this one as well I think, so it could be user provided

Reply:
It's there, it's just a list now https://github.com/Netflix/dgs-framework/pull/2167/files#diff-81c22179fe1aa8884cdd1f6e458ea9be32c11bff73c59367c6d02bb256af8b47R115